ui/providers/BottomDrawer/containers/LogViewer/__tests__/LogViewer.test.tsx

```
@@ -79,6 +79,8 @@ jest.mock('../LogViewerToolbar', () => {
         <button data-testid="toggle-wrap" onClick={props.onToggleWrap}>wrap</button>
         <button data-testid="toggle-follow" onClick={props.onToggleFollow}>follow</button>
         <button data-testid="toggle-paused" onClick={props.onTogglePaused}>pause</button>
+        <button data-testid="copy-visible" onClick={props.onCopyVisible}>copy visible</button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)